src/index.js

```diff
@@ -1 +1,76 @@
 'use strict';
+import 'dotenv/config';
+import express from 'express';
+import cors from 'cors';
+import { Console } from './utils/Console.js';
+import { authRouter } from './routes/authRouter.js';
+import cookieParser from 'cookie-parser';
+import { chatRouter } from './routes/chatRouter.js';
+import { WebSocketServer } from 'ws';
+// import { EventEmitter } from 'events';
+// import { chatService } from './service/chat.service.js';
+
+const PORT = process.env.PORT || 3005;
+const app = express();
+// const emitter = new EventEmitter();
+
+app.use(
+  cors({
+    origin: process.env.CLIENT_HOST,
+    credentials: true,
+  }),
+);
+app.use(express.json());
+app.use(cookieParser());
+
+app.get('/', (req, res) => {
+  res.send('Welcome');
+});
+
+app.use('/auth', authRouter);
+app.use('/chat', chatRouter);
+
+const server = app.listen(PORT, () => {
+  Console.log(`Server is running ${PORT}`);
```

Comment:
The console log message could be improved for clarity. Consider changing it to `Server is running on port ${PORT}`.